Owncloud iOs Client/Login/Login/UniversalLoginViewController.swift

```diff
@@ -933,24 +922,31 @@ public enum TextfieldType: String {
                     
                 } else {
 
+                    //1.Generate correct user
+                    if self.loginMode == .create {
```

Comment:
this could be changed to a switch. also you can include _forceAccountMigration_ in the enum as .forceAccountMigration. This way you can extrac into a function the methods inside the second if and create a more clean switch.